assets/src/index.css

```diff
@@ -26,6 +26,11 @@ code {
   animation: sk-scaleout 1.0s infinite ease-in-out;
 }
 
+/* Modifies default focus style for header tag in ViewHeader component */
+.view-header:focus {
+  outline-offset: 5px;
```

Comment:
I don't see the effect of this styling in the UI. I have changed the outline-offset setting am I missing something?
![Screen Shot 2020-06-05 at 11 04 41 AM](https://user-images.githubusercontent.com/8579775/83892412-a0633300-a71c-11ea-9b5a-899f0addc32b.png)

Reply:
The `outline-offset: 5px;` property doesn't see the effect the chrome, but only in FF. I am not sure why is this case? Need to support all prime browser as per this [list](https://github.com/tl-its-umich-edu/my-learning-analytics/wiki/MyLA-Browser-Support)

Reply:
```
outline-offset: 5px;
  outline-style: solid;
```
This combination seem to work in Chrome to add little spacing around the box

Reply:
Okay, I think I added a specific style for the focus so it is consistent across browsers. Below is a screenshot of how it looks in Chrome. I opted to use the `secondary` color (`#2C6496`), since that indicates a selected state in the rest of the application.
![Screen Shot 2020-06-05 at 1 24 48 PM](https://user-images.githubusercontent.com/35741256/83905751-33599880-a730-11ea-9623-42c994746083.png)

Reply:
@jennlove-um, I would like your input on how this looks. What I've seen online suggests that [focus should be visible](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html), but this could be considered a slightly different case since it is not for an element that accepts user interaction. Gonzalo said that, if we are concerned about the appearance, it would be okay to remove the focus style entirely. It will still focus when the page loads, but it just won't be visible. Note that the focus box (seen above) will disappear if the user presses Tab or otherwise moves the focus elsewhere.